controllers/vrg_volrep.go

```diff
@@ -1877,6 +1878,8 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 		// Restore all PVs found in the s3 store. If any failure, the next profile will be retried
 		pvCount, err = v.restorePVsFromObjectStore(objectStore, s3ProfileName)
 		if err != nil {
+			v.log.Error(err, fmt.Sprintf("error fetching PV cluster data from S3 profile %s", s3ProfileName))
```

Comment:
Based on the failing function name, this should be: "cannot restore PVs from s3 store $q"

Since we retry using the next profile, is this really an error or we can use INFO level message?

(We should really use WARNING log level but the logger we used does not have that)